drive-contents/hardware/drivers.py

```diff
@@ -118,8 +121,21 @@ def init_sensors():
         haptic_enable.switch_to_output()
 
         _enable_haptic()
-        # haptic = DRV2605(i2c, address=0x5E)
-        haptic = DRV2605(i2c, address=0x5A)
+
+        # determine the current I2C device for the haptic devices
+        # scan for what adresses are on the bus
+        if i2c.try_lock():
+            addrs = i2c.scan()
+            i2c.unlock()
+        else:
+            raise RuntimeError(f"unable to scan i2c bus")
+
+        if 0x5E in addrs:
+            haptic = DRV2605(i2c, address=0x5E)
+        elif 0x5A in addrs:
+            haptic = DRV2605(i2c, address=0x5A)
+        else:
+            raise RuntimeError(f"unable to find haptic drive on the i2c bus")
```

Comment:
I anticipate this being a relatively common error for end-users to have. Maybe include a description or troubleshooting in a help document.